01-data-structures/02-stacks-and-queues/myqueue/myqueue_spec.rb

```diff
@@ -6,26 +6,15 @@
   let(:q) { MyQueue.new }
 
   describe "#enqueue" do
-
-    it "updates the head and tail accessors properly" do
+    it "enqueues strings to the queue, verifies with head and tail" do
```

Comment:
Ahh, gotcha. So here, the semantics of the test name is not correct because the intention is not clear.

The code within the test itself is good, as long as this is changed to something like "enqueues strings to queue"